DataRepo/templates/DataRepo/search_advanced.html

```diff
@@ -236,58 +247,56 @@ <h3>{{ cur_res.count|default:"0" }} Records</h3>
-                                        {{ pd.peak_group.msrun.sample.animal.name }}
-                                    </a>
-                                </td>
-
-                                <!-- Infusate -->
-                                <td>
-                                    {{ pd.peak_group.msrun.sample.animal.tracer_compound.name }}
-                                </td>
-
-                                <!-- Compound -->
-                                <td>
-                                    <a href="{% url 'sample_detail' pd.peak_group.id %}">
-                                        {{ pd.peak_group.name }}
+                    {% for pd in res.all %}
+                        <tr>
+                            <!-- Sample -->
+                            <td>
+                                <div class="nobr">
+                                    <a href="{% url 'sample_detail' pd.peak_group.msrun.sample.id %}">
+                                        {{ pd.peak_group.msrun.sample.name }}
                                     </a>
-                                </td>
-
-                                <!-- Atom Label -->
-                                <td>
-                                    {{ pd.labeled_element }}:{{ pd.labeled_count }}
-                                </td>
-
-                                <!-- Corrected Abundance -->
-                                <td class="text-end">
-                                    <p title="{{ pd.corrected_abundance }}">{{ pd.corrected_abundance|floatformat:1 }}</p>
-                                </td>
-
-                                <!-- Fraction -->
-                                <td class="text-end">
-                                    <p title="{{ pd.fraction }}">{{ pd.fraction|floatformat:4 }}</p>
-                                </td>
-                            </tr>
-                        {% endfor %}
+                                </div>
+                            </td>
+
+                            <!-- Tissue -->
+                            <td>
+                                {{ pd.peak_group.msrun.sample.tissue.name }}
+                            </td>
+
+                            <!-- Animal -->
+                            <td>
+                                <a href="{% url 'animal_detail' pd.peak_group.msrun.sample.animal.id %}">
+                                    {{ pd.peak_group.msrun.sample.animal.name }}
+                                </a>
+                            </td>
+
+                            <!-- Infusate -->
+                            <td>
+                                {{ pd.peak_group.msrun.sample.animal.tracer_compound.name }}
+                            </td>
+
+                            <!-- Compound -->
+                            <td>
+                                <a href="{% url 'sample_detail' pd.peak_group.id %}">
```

Comment:
NB: This should be a link to the PeakGroup
```suggestion
                                <a href="{% url 'peak_group_detail' pd.peak_group.id %}">
```

Reply:
Good catch.

Reply:
- [ ] Fix PG link in template

Reply:
Fixed in both output formats

Reply:
Incidentally, the name for the url pattern for the peakgroup detail page is `peakgroup_detail`.  It's reassuring to know that the tests caught this.